From 072c62f2059dd086266d4c74de25591836da6d27 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Nov=C3=BD?= <onovy@debian.org>
Date: Sat, 5 Aug 2017 16:21:49 +0200
Subject: Fix test for Python 3.6

---
 tests/test_version_table.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_version_table.py b/tests/test_version_table.py
index 92cb447..45a30df 100644
--- a/tests/test_version_table.py
+++ b/tests/test_version_table.py
@@ -35,8 +35,8 @@ class TestMigrationContext(TestBase):
         self.transaction = self.connection.begin()
 
     def tearDown(self):
-        version_table.drop(self.connection, checkfirst=True)
         self.transaction.rollback()
+        version_table.drop(self.connection, checkfirst=True)
         self.connection.close()
 
     def make_one(self, **kwargs):
